scout/server/config.py

```diff
@@ -18,7 +18,9 @@
 
 # connection details for LoqusDB MongoDB database
 # LOQUSDB_SETTINGS = dict(
-#     database='loqusdb',
+#     binary='/path/to/bin/loqusdb',
```

Comment:
Is this really better than a separate loqus REST API app, with its own config and environment etc? You have thought more about this - and I do like a good CLI, so I'll easily accept your argument. Just curious why this when so much else in the archipelago is REST-ish micro apps?

Reply:
I'm not sure how that would work in this case since loqusdb is not a server. But my knowledge of REST APIs is very limited. At least this thing works and it is a nice way to do it when packages are in different environments

Reply:
Ok, lets merge this now and test it out! You have already put in the work, and it serves the isolation point well. Remind us and we will help out with a simple REST-interface for loqusdb! That would be a nice addition! Actually, I'm submitting an issue. :wink: